src/FTPClient.jl

```diff
@@ -16,6 +16,7 @@ type FTPClientError <: Exception
        lib_curl_error::UInt32
 end
 
+@enum FTP_MODE ascii_mode binary_mode
```

Comment:
Hmm I don't like that you have to know `FTP_MODE` and `TRANSFER` to be able to use it, and the fact that you have to use a `module` instead of a `baremodule` makes it less efficient and clean. The existing way using `@enum` is fine for now I guess but I'll think about this.